src/components/App/App.css

```diff
@@ -53,3 +53,16 @@ img {
   box-shadow: 0 0 0.5rem 3px black;
   background-image: url(/public/assets/images/space-bg.jpg);
 }
+
+.fade-in {
+	animation: fade-in 1.5s cubic-bezier(0.390, 0.575, 0.565, 1.000) both;
```

Comment:
⛏️ This looks great! Maybe consider a linter to automatically format your files and make sure that your tabbing is always consistent!